### DISCUSSION TITLE
Resources which can help implement emulation of the Ensoniq SoundScape

### DISCUSSION THREAD

**Discussion by Ompronce** [86Box/86Box#1720] [General]
**Is your feature request related to a problem? Please describe.**
I have found a website which has datasheets and technical information for various Ensoniq synthesizers, it notably has full register documentation for all of the parts used on the Ensoniq SoundScape ISA cards.

**Describe the solution you'd like**
The resources I'm linking to can help more experienced contributors implement an emulation of the Ensoniq SoundScape sound cards.

**Describe alternatives you've considered**
I haven't seen or considered any alternative implementations as of the time of writing, although MAME emulates the OTTO synth used in these cards. 86Box emulates the Gravis Ultrasound, which is actually based on the Ensoniq OTTO chipset.

**Additional context**
Website containing datasheets and more information - http://zine.r-massive.com/ensoniq-technical-documents-and-schematics/ (The ODIE, OTTO and ESP datasheets are the ones most relevant to SoundScape emulation, the ESP effects DSP was only used in the SoundScape Elite card)

Website containing instructions on how to dump the sound ROM from an Ensoniq SoundScape - http://www.os2museum.com/wp/dumping-ensoniq-soundscape-roms/

Website containing info on the SoundScape Vivo, which does not use a M68K CPU on board and basically is an ISA-based AudioPCI except with the OTTO synth and ROM on board - https://www.os2museum.com/wp/any-soundscape-vivo-fans-here/

**Comment by OBattler** [comment]
Closing because I moved it to discussions: https://github.com/86Box/86Box/discussions/1717 .
